Retire fix1571 Amendment

pratikmankawde opened the pull request:
## High Level Overview of Change
This change retires the `fix1571` amendment.

### Context of Change
Original amendment: https://xrpl.org/resources/known-amendments#fix1571
Amendments activated for more than 2 years can be retired.

### Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (non-breaking change that only restructures code)
- [ ] Performance (increase or change in throughput and/or latency)
- [ ] Tests (you added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation update
- [ ] Chore (no impact to binary, e.g. `.gitignore`, formatting, dropping support for older tooling)
- [ ] Release

pratikmankawde commented on src/test/app/Escrow_test.cpp:
```diff
@@ -294,74 +294,51 @@ struct Escrow_test : public beast::unit_test::suite
+        env(escrow::finish("carol", "alice", seqFt), fee(150 * baseFee));
+        BEAST_EXPECT(
+            env.balance("bob") ==
+            XRP(5200));  // 5100 (from last transaction) + 100
```
@ximinez Can you please confirm if this new unit test which checks presence of `cancel time and finish time` combination is correct?

vvysokikh1 replied:
considering there's a bunch of tests in this file that have almost same check (search for "Metadata to self" testcase for example), I'd say this seems correct

pratikmankawde replied:
Thanks for confirming.

vvysokikh1 reviewed (approved):
lgtm

a1q123456 reviewed (approved):
LGTM

bthomee committed:
refactor: Retire fix1571 amendment

Amendments activated for more than 2 years can be retired. This change retires the fix1571 amendment.
